archive/app/ui/NotFound.scala

```diff
@@ -1,14 +1,9 @@
 package ui
 
-import conf.Configuration
-import play.api.libs.json.{JsValue, Json}
 import rendering.Renderable
+import rendering.core.JavascriptProps
+import play.api.libs.json.{JsValue, Json}
 
 object NotFound extends Renderable {
-  override def props: Option[JsValue] = Some(
-    Json.obj(
-      "beaconUrl" -> Configuration.debug.beaconUrl
-    )
-  )
+  override def props: Option[JsValue] = Some(Json.toJson(JavascriptProps()))
```

Comment:
The default `JavascriptProps` are automatically added [here](https://github.com/guardian/frontend/pull/17995/files#diff-68c19301733dc555c0f2aa32c2a3b32dR22) 
Do we need to specify them here?